tests/unit/spec/base/mozilla-fxa.js

```diff
@@ -9,21 +9,21 @@ describe('mozilla-fxa.js', function() {
     describe('FxaState.applyStateToBody', function() {
 
         beforeEach(function() {
-            $('body').attr('class', 'js state-fxa-default');
+            document.body.setAttribute('class', 'js state-fxa-default');
```

Comment:
Would swapping `setAttribute` for `classList.add` make more sense here?

Reply:
Good suggestion 👍